src/layers/domain/api/sds/query.py

```diff
@@ -45,3 +45,31 @@ def check_filters(cls, values: dict):
 
     def get_non_null_params(self):
         return self.dict(exclude_none=True)
+
```

Comment:
Think this needs to go in it's own file outside of SDS as this folder and file will eventually get removed